### PR TITLE
Bottom navigation

### LINKED ISSUE #12: Bottom Navigation
Create a bottom nav panel for phones and tablets to route to coaches, managers and admins dashboards

### PR DESCRIPTION
**Pull request by Papajmil**
Created a bottom navigation bar that shows options based on users authentication. It also uses mdi icons and removes words when the window is small enough (works with phones and tablets).

closes #12

### CODE REVIEW

File: src/components/BottomNav.vue [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<template>
+  <div>
+    <div>Active: {{ bottomNav }} </div>
+    <div>Coach: {{ coach }} </div>
+    <div>Manager: {{ manager }} </div>
+    <div>Admin: {{ admin }} </div>
+    <v-bottom-nav
+      :active.sync="bottomNav"
+      :value="true"
+      fixed
```
Comment (fezproof) [inline src/components/BottomNav.vue]:
```suggestion
      app
```
Should use app not fixed

File: src/components/BottomNav.vue [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    >
+      <v-btn
+        v-if="coach"
+        color="teal"
+        flat
+        value="Coach Page"
+      >
+        <span v-show="$vuetify.breakpoint.mdAndUp">Coaches</span>
```
Comment (fezproof) [inline src/components/BottomNav.vue]:
```suggestion
        <span v-show="$vuetify.breakpoint.smAndUp">Coaches</span>
```
Should be sm and up due to tablets

File: src/components/BottomNav.vue [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        <v-icon>mdi-football</v-icon>
+      </v-btn>
+
+      <v-btn
+        v-if="manager"
+        color="teal"
+        flat
+        value="Regional Manager Page"
+      >
+        <span v-show="$vuetify.breakpoint.mdAndUp">Regional Managers</span>
```
Comment (fezproof) [inline src/components/BottomNav.vue]:
```suggestion
        <span v-show="$vuetify.breakpoint.smAndUp">Coaches</span>
```
Should be sm and up due to tablets

File: src/components/BottomNav.vue [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        <v-icon>mdi-sitemap</v-icon>
+      </v-btn>
+
+      <v-btn
+        v-if="admin"
+        color="teal"
+        flat
+        value="Admin Page"
+      >
+        <span v-show="$vuetify.breakpoint.mdAndUp">Administrators</span>
```
Comment (fezproof) [inline src/components/BottomNav.vue]:
```suggestion
        <span v-show="$vuetify.breakpoint.smAndUp">Coaches</span>
```
Should be sm and up due to tablets

**Comment by Papajmil** [comment]
I've made the changes and committed to the branch

**Pull request merged**